README.md

```diff
@@ -100,14 +100,40 @@ or
   git push -u origin main
 ```
 
-If you get an error message like the one below, you probably forgot to fork the repository before cloning it. It is best to start over and fork the project repository first.
+If you get an error message like the one below, you probably forgot to fork the repository before cloning it.
 
 ```bash
 ERROR: Permission to EddieHubCommunity/hacktoberfest-practice.git denied to <your-github-username>.
 fatal: Could not read from remote repository.
 Please make sure you have the correct access rights and the repository exists.
 ```
 
+No need to panic or loose your streak at this point. Follow along to fix this issue.
+
+To fix this issue, fork the project now using the gray `Fork` button in the top right of this page.
+Once the fork is complete it will redirect you to the newly forked repo on your github account.
+Run these commands at this point and you are good to continue:
+
+```
+git remote remove origin
+git remote add origin git@github.com:<your github usename>/hacktoberfest-practice.git
+```
+
+You can confirm that your local copy is pointing to forked repo under your account by running this command
+
+```
+git remote -vv
+```
+
+output should be like below:
```

Comment:
I think we could improve the wording here. What do you think?


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
Nothing that a developer should not already be understanding from this working.